public/docs/content/.pages.yml

```diff
@@ -1,7 +1,8 @@
 ---
 nav:
-  - Welcome: welcome
+  - Overview: overview
+  - Getting started: getting_started
   - Developer Guides: developer_guides
-  - Walkthroughs: walkthroughs
   - API Reference: api
-  - FAQ: faqs
+  - Release Notes: release_notes
```

Comment:
The content for this should link to: https://github.com/aws/aws-prototyping-sdk/releases

Reply:
Updated

Reply:
resolved